modules/recommended-scale-practices.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-cluster-scaling-practices.adoc
+
+[id="recommended-scale-practices_{context}"]
+= Recommended practices for scaling the cluster
+
+When scaling up the cluster to higher node counts, spread the nodes across all
+of the available zones for higher availability. The controller might not be able
+to create the machines if the replicas in the MachineSets are set to higher
+numbers all at one time. The number of requests the cloud platform, which
+{product-title} is deployed on top of, is able to handle impacts the process.
+The controller will start to query more while trying to create, check, and
+update the machines with the status. The cloud platform on which {product-title}
+is deployed has API request limits and excessive queries might lead to machine
+creation failures due to cloud platform limitations. Therefore, it is
+recommended that you scale up the cluster by 25 to 50 nodes at a time in order
+to reach higher node counts.
+
+It is also recommended to enable machine health checks when scaling to large node
```

Comment:
Enable machine health checks when scaling to large node counts. In case of failures, health checks automatically repair unhealthy machines in a particular MachinePool.